server/test/user.spec.js

```diff
@@ -101,6 +102,32 @@ describe('Users Routes', () => {
       expect(res).to.be.a('object');
       expect(res.body).to.have.property('data');
     });
+
+    it('should not retrieve users functionality if authorization token is not valid', async () => {
```

Comment:
Neat job. Consider tweaking the message in this test. I don't think you need the 'functionality' term.

Reply:
Can you explain this more

Reply:
You could simply put 'should not retrieve users if authorization token is invalid'

Reply:
Ok but it would not be descriptive enough

Reply:
Actually it is. You aim is to retrieve users right? And a person who is not logged can't retrieve users. Or if you are referring to the function that retrieves users, then use this:
'should not call the retrieve users function if authorization token is invalid'

Reply:
Aim of this feature is to list users functionality (profile)

Reply:
Hi @geekdanitek, please what exactly do you mean by users functionality?

Reply:
I think the description is suitable. He is trying to retrieve user functionality.

Reply:
Got it.